src/transaction/log_tran_table.c

```diff
@@ -4938,41 +4906,41 @@ logtb_update_global_unique_stats_by_delta (THREAD_ENTRY * thread_p, BTID * btid,
   if (log)
     {
       RECDES undo_rec, redo_rec;
-      char undo_rec_buf[(3 * OR_INT_SIZE) + OR_BTID_ALIGNED_SIZE + BTREE_MAX_ALIGN], *datap = NULL;
-      char redo_rec_buf[(3 * OR_INT_SIZE) + OR_BTID_ALIGNED_SIZE + BTREE_MAX_ALIGN];
+      char undo_rec_buf[(3 * OR_BIGINT_SIZE) + OR_BTID_ALIGNED_SIZE + BTREE_MAX_ALIGN], *datap = NULL;
+      char redo_rec_buf[(3 * OR_BIGINT_SIZE) + OR_BTID_ALIGNED_SIZE + BTREE_MAX_ALIGN];
 
       /* although we don't change the btree header, we still need to log here the new values of statistics so that they
        * can be recovered at recover stage. For undo purposes we log the increments. */
       undo_rec.data = NULL;
-      undo_rec.area_size = 3 * OR_INT_SIZE + OR_BTID_ALIGNED_SIZE;
+      undo_rec.area_size = 3 * OR_BIGINT_SIZE + OR_BTID_ALIGNED_SIZE;
       undo_rec.data = PTR_ALIGN (undo_rec_buf, BTREE_MAX_ALIGN);
 
       undo_rec.length = 0;
       datap = (char *) undo_rec.data;
       OR_PUT_BTID (datap, btid);
       datap += OR_BTID_ALIGNED_SIZE;
-      OR_PUT_INT (datap, null_delta);
-      datap += OR_INT_SIZE;
-      OR_PUT_INT (datap, oid_delta);
-      datap += OR_INT_SIZE;
-      OR_PUT_INT (datap, key_delta);
-      datap += OR_INT_SIZE;
+      OR_PUT_BIGINT (datap, &null_delta);
```

Comment:
@byungwook-kim 
I left a comment regarding order of packs/unpacks in issue - http://jira.cubrid.org/browse/CBRD-24110 .